plugins/com.google.cloud.tools.eclipse.appengine.libraries/plugin.xml

```diff
@@ -66,8 +66,8 @@
     </library>
 
     <library
-          id="servlet-api-2.5"
-          name="Servlet API 2.5"
+          id="appengine-servlet-2.5"
+          name="App Engine: Servlet API 2.5"
```

Comment:
Why change the name? It's not app engine specific.

Reply:
The intent is for this library to be a container of dependencies.  Maybe `aes-jre7` / _App Engine standard environment for Java 7_?

Reply:
I originally had combined the Servlet, JSP, and JSTL+EL deps into `<library>` elements for a Servlet 2.5 container, and a Servlet 3.1 container, and kept the `appengine-api` separate with them all included by the `ServletClasspathContainer`.  But this didn't make sense since it's up to the actual container implementation as to whether it includes implementations of JSP, JSTL, EL.

(Aside, perhaps it shouldn't the library definitions that decide whether they should be 'exported' or not.)

Reply:
Interesting. Where should we indicate whether they're exported?

Reply:
If we name it "App Engine: Servlet API 2.5" (as it stands in the PR) implying this `<library>` is specifically for App Engine Java 7, I think we are good; no need to think about if JSTL+EL is supported in a different container or if they are exported.

Reply:
These libraries are provided by the servlet container, so they don't need to be exported, and they won't be included in the resulting .war.